bio/sexdeterrmine/wrapper.py

```
@@ -0,0 +1,44 @@
+import os.path
+
+log = snakemake.log_fmt_shell(
+    stdout=False,
+    stderr=True,
+    append=True,
+)
+extra = snakemake.params.get("extra", "")
+
+with TemporaryDirectory() as tempdir:
+    old_path = os.getcwd()
+    depth_path = os.path.realpath(snakemake.input.depth)
```

Ruff (0.7.0)

Undefined name `snakemake`

(F821)